source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -100,13 +100,13 @@ If you need help along the way, visit the [Ember Community chat and forums](http
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Octane at EmberConf 2020 📢](https://emberconf.com)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+With [Ember Octane](https://blog.emberjs.com/2019/12/20/octane-is-here.html) released it is not a surprise that it’s also a hot topic for [EmberConf 2020](https://emberconf.com/#/speakers). 
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+If you want to [listen and talk to people](https://emberconf.com/#/speakers) about Octane, EmberConf in Portland, OR is the place to be. The 16th till 18th of March will be filled with awesome talks by [Godfrey Chan](https://github.com/chancancode), [Suchita Doshi](https://github.com/suchitadoshi1987), [DanMonroe](https://github.com/DanMonroe) and many others. 
+
+It’s of course also a nice place to meet up with your Ember-friends and learn about many more topics. For more information, check the [EmberConf website](https://emberconf.com) and join the #ember-conf channel on Discord.
```

Comment:
Regarding Alex linting, I think we can omit the phrase "of course" here, and the sentence will still flow nicely and sounds friendly. Would that be okay with you?
```suggestion
It’s also a nice place to meet up with your Ember-friends and learn about many more topics. For more information, check the [EmberConf website](https://emberconf.com) and join the #ember-conf channel on Discord.
```